core/src/test/java/org/frankframework/mongodb/MongoDbSenderTest.java

```diff
@@ -221,6 +225,7 @@ void testFindManyUsingParameter() throws Exception {
 
 	@Test
 	void testFindManyCountOnly() throws Exception {
+		insertStudentRecord();
```

Comment:
Note: The previous unit tests were depending on each other and needed the right order of execution. Now, they also can be ran independently.